src/Transformers/StopWordFilter.php

```diff
@@ -18,24 +18,35 @@
  */
 class StopWordFilter extends RegexFilter
 {
+    public const DEFAULT_EN = [
+        'a', 'an', 'and', 'are', 'as', 'at', 'be', 'but', 'by', 'for', 'if', 'in', 'into', 'is', 'it',
+        'no', 'not', 'of', 'on', 'or', 'such', 'that', 'the', 'their', 'then', 'there', 'these', 'they',
+        'this', 'to', 'was', 'will', 'with',
+    ];
+
     /**
      * @param string[] $stopWords
      * @throws \InvalidArgumentException
      */
-    public function __construct(array $stopWords = [])
+    public function __construct(array $stopWords = self::DEFAULT_EN)
```

Comment:
Hmm, I may have left this in by accident. I'm not sure if this is a BC break? Previously providing an empty array would have produced an empty `$patterns` array causing the parent class to throw an exception. For increased clarity I moved that exception to this class, but am not sure if we should set a non empty default (although you may feel that this is a nice developer experience?)

Happy to revert this line if you want me to 👍

Reply:
The reason we have not included a list of default stop words is because they are highly language and context dependent. In addition, default stop words often decrease the accuracy of a mode if applied blindly. Let's let the user come up with their own stop words based on their language and problem type.